src/screens/augmentedReality/ArtworkDetailScreen.js

```diff
@@ -16,28 +16,16 @@ import {
 } from '../../components';
 import { texts } from '../../config';
 import { checkDownloadedData, downloadObject, DOWNLOAD_TYPE } from '../../helpers';
-import { usePullToRefetch, useStaticContent } from '../../hooks';
 import { ScreenName } from '../../types';
 
 export const ArtworkDetailScreen = ({ route, navigation }) => {
-  const {
-    data: artworkDetail = '',
-    error,
-    loading,
-    refetch
-  } = useStaticContent({
-    type: 'html',
-    name: 'artworkDetail'
-  });
-
   const [isModalVisible, setIsModalVisible] = useState(false);
   const [data, setData] = useState(route?.params?.data ?? []);
-  const [isLoading, setIsLoading] = useState(loading);
+  const [isLoading, setIsLoading] = useState(true);
   const index = route?.params?.index;
+  const description = data?.[index]?.description;
```

Comment:
```
const description = data?.[index]?.description;
const { downloadType } = data[index]?.payload;
```

why are you checking the presence of data for description now? for payload this is not the case.